### PR TITLE
Tests for ObjectInfo class

### PR DESCRIPTION
**Pull request by makiam**

### CODE REVIEW

File: Tests/src/artofillusion/object/ObjectInfoTest.java [hunk 1]
```diff
@@ -0,0 +1,852 @@
...
+    assertTrue(target.getDistortion() instanceof DistortionImpl);
+    
+  }
+  
+  /**
+   * Test objectInfo copy data from other objectInfo and points to same geometry
+   * Checks that empty distortion data overwrites existed one
+   */ 
+  @Test
+  public void testCopyInfoWithNullDistortionOverExisted()
```
Comment (Sailsman63) [inline Tests/src/artofillusion/object/ObjectInfoTest.java]: This test fails with the current codebase. There is an explicit check to make sure that the distortion is _not_ null before copying it over existing data. See:

https://github.com/ArtOfIllusion/ArtOfIllusion/blob/7623fc4a7d3d58434c90b294ad499479f210d55c/ArtOfIllusion/src/artofillusion/object/ObjectInfo.java#L136-L139

@peastman do you know what the reason is here for _not_ overwriting with a null?

Comment (peastman) [reply Tests/src/artofillusion/object/ObjectInfoTest.java]: No reason I can think of.  I suspect I just forgot to handle the case where it's null.

Comment (Sailsman63) [reply Tests/src/artofillusion/object/ObjectInfoTest.java]: Makes sense to me.

**Comment by Sailsman63** [comment]
Thanks for the tests and quick fix!

**Pull request merged**